etc/testing/console-test-helm-values/pachdev/pach-local-default-pachyderm-values.json

```diff
@@ -0,0 +1,16 @@
+{
+  "deployTarget": "custom",
+  "console": {
+    "enabled": true
+  },
+  "oidc": {
+    "issuerURI": "http://pachd:30658/dex"
+  },
+  "pachd": {
+    "clusterDeploymentID": "enterprise",
+    "lokiDeploy": false,
+    "lokiLogging": false,
+    "resources": null,
+    "rootToken": "pizza"
```

Comment:
We should update our e2e tests to use the default root token that pachyderm uses.